.github/workflows/ci.yaml

```diff
@@ -446,6 +446,77 @@ jobs:
         run: |
           poetry run pytest tests/ -vvv -n 2 ${PYTEST_OPTS}  ${{ matrix.PYTEST_OPTS }}
 
+  min-btc-support:
+    name: Test BTC version 25.0 ${{ matrix.name }}
```

Comment:
note that the `matrix.name` here is "clang" (see L462). Not sure if that's very useful info to include in the test name?

would be more useful to change the `25.0` to use the parameterized bitcoin version instead?

for example

```
name: Test min-bitcoind v${{ matrix.bitcoin_version }}
```

Reply:
I added the matrix.name in the title to clearly show that the step has been tested with `clang` only. But I have updated the title to be more clearer with `Test minimum supported BTC v${{ matrix.MIN_BTC_VERSION }} with ${{ matrix.NAME }}`.